lib/src/native/realm_core.dart

```diff
@@ -648,6 +685,27 @@ class _RealmCore {
     });
   }
 
+  RealmObjectHandle? findExisting(Realm realm, int classKey, RealmObjectHandle other) {
+    final key = _realmLib.realm_object_get_key(other._pointer);
+    final pointer = _realmLib.invokeGetPointer(() => _realmLib.realm_get_object(realm.handle._pointer, classKey, key));
+    return RealmObjectHandle._(pointer);
+  }
+
+  void renameProperty(Realm realm, String objectType, String oldName, String newName, SchemaHandle schema) {
+    using((Arena arena) {
+      _realmLib.invokeGetBool(() => _realmLib.realm_schema_rename_property(
+          realm.handle._pointer, schema._pointer, objectType.toCharPtr(arena), oldName.toCharPtr(arena), newName.toCharPtr(arena)));
+    });
+  }
+
+  bool removeType(Realm realm, String objectType) {
```

Comment:
```suggestion
  bool deleteType(Realm realm, String objectType) {
```
as per the above suggestion

Reply:
This got resolved but the method is the same. Should we rename to `deleteType`?

Reply:
Renamed the public API but forgot the Core API, will fix.